Add-ons/UmbracoCourier/Installing/index.md

```diff
@@ -0,0 +1,66 @@
+# Installation
+
+## Installing Courier
+
+The following guide will use two example sites; *development.site* and *live.site*. The guide will go through how to install and configure Umbraco Courier on two sites in order to be able to use Courier to transfer types and content between the two.
+
+* Go to the **Developer** section of the backoffice
+* Find **Packages** in the left navigation tree
+* The **Umbraco Courier** package is usually one of the top packages - otherwise you can simply search for it
+* Follow the installation directions
+* Once Courier has been installed, make sure to **refresh the page**
+* Next step is to **add a location** 
+    * In the gif below Courier is installed on development.site, and live.site is added as the location
+* The location you add, will be set in the `config/courier.config` file as a *repository*. Learn more about this in the [Repository Providers](../Configuration/RepositoryProviders.md) article.
+
+![InstallingCourier](images/InstallCourier.gif)
+
+Go through these steps on all sites where you want to use Courier.
+
+Before you can start transferring between your sites, you need to **add a Courier API key** to the `courier.config` file on all the sites.
+
+* Find the `courier.config` file in the `/config` folder
+* In the `<security>` section, find the `<auth>` section - it will look like this:
+
+```xml
+<auth>
+    <method>credentials</method>
+    <apikey></apikey>
+</auth>
+```
+
+* Change `<method>` to **token**
+* Add a **randomly generated api key with at least 10 characters** - this could be anything, fx 1234567890
```

Comment:
Can it be spaces, letters, symbols, greek letters, smilies? May need to be more specific here..

Reply:
`<apikey>🤔😄😁🙂👍😉😃😬😕🦄</apikey>` 😄😄

Reply:
🙄